server/formatio/omv.py

```diff
@@ -107,7 +108,9 @@ def read(data, path):
         meta_dataset = metadata['dataSet']
 
         for meta_column in meta_dataset['fields']:
-            data.dataset.append_column(meta_column['name'])
+            name = meta_column['name']
+            import_name = meta_column.get(name)
```

Comment:
my feeling is that this wants to be:

`import_name = meta_column.get('importName', name)`